pywren/scripts/pywrencli.py

```diff
@@ -204,8 +205,11 @@ def deploy_lambda(ctx, update_if_exists = True):
     zipfile_obj = zipfile.ZipFile(file_like_object, mode='w')
 
     # FIXME see if role exists
-    files = glob2.glob(os.path.join(SOURCE_DIR, "../**/*.py"))
-    for f in files:
+    module_dir = os.path.join(SOURCE_DIR, "../")
+    
+    for f in ['wrenutil.py', 'wrenconfig.py', 's3util.py', 'wrenhandler.py', 
+              'version.py', 'jobrunner.py', 'wren.py']:
```

Comment:
I feel it could be nice to still compare with files in the directory and output a warning message if there is a mismatch. Without which we might leave some hard-to-detect bugs when we add/remove files in the future.